app/models/event.rb

```diff
@@ -0,0 +1,10 @@
+class Event < ApplicationRecord
+
+    scope :upcoming, -> { where("event_date >= ?", Date.today) }
+    scope :past, -> { where("event_date < ?", Date.today) }
+
+
+    belongs_to :creator, class_name: "User"
+    has_many :attended_events, foreign_key: "attended_event_id"
+    has_many :attendees, through: :attended_events, source: :attendee
```

Comment:
Why do we still need to use `source` here when the association name is already `attendee` ? We only use `source` when our association name doesn't correspond to a named column in the table